followthemoney/schema/Interval.yaml

```diff
@@ -14,6 +14,12 @@ Interval:
     An object which is bounded in time.
   matchable: false
   abstract: true
+  temporalExtent:
+    start:
+      - startDate
+      - date
+    end:
+      - endDate
```

Comment:
Initially, I had planned to use a property as the start *and* end in case of properties that don’t have a corresponding property that can represent the end.

For example, in case of an `Event` with just a `date` (and no `startDate`/`endDate`), it makes sense semantically to assume that it’s a single-day event and thus the `date` is both the start and end.

This doesn’t work well in all cases though, so I decided to only set the temporal extent if a property clearly represents the end of a range (for example `endDate`, `dissolutionDate`).

Reply:
+1 on using convention on the edge case :)